internal/server/storage/autoswitch.go

```diff
@@ -0,0 +1,115 @@
+package storage
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog"
+
+	"github.com/npavlov/go-metrics-service/internal/domain"
+	"github.com/npavlov/go-metrics-service/internal/model"
+)
+
+// AutoSwitchRepo monitors the database connection and provides the appropriate storage.
+type AutoSwitchRepo struct {
```

Comment:
[CHANGE REQUEST] We should not switch between db and memory repos automatically.

Reply:
[CHANGE REQUEST] We should have only one repo

Reply:
okay, so we set one at the start